test/token-assert.js

```diff
@@ -564,6 +564,38 @@ describe('modules/token-assert', function() {
 
                 assert(!onError.called);
             });
+
+            it('should edit the whitespaceBefore', function() {
+                var file = createJsFile('x\n=y;');
+
+                var tokenAssert = new TokenAssert(file);
+                tokenAssert.on('error', function() {});
```

Comment:
It throws an exception if I don't listen to the event. That is why I'm simply adding the function.

> TypeError: Uncaught, unspecified "error" event.